mod-users/src/main/resources/domain/mod-users/users-junit.feature

```diff
@@ -0,0 +1,28 @@
+Feature: mod-users integration tests
+
+  Background:
+    * url baseUrl
+    * table modules
+      | name                      |
+      | 'mod-login'               |
+      | 'mod-permissions'         |
+      | 'mod-users'               |
+      | 'mod-circulation-storage' |
```

Comment:
Why is mod-circulation-storage needed for a users test?

Reply:
i will correct it . thankyou